src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -647,12 +650,27 @@ private ImmutableArray<ParameterSymbol> ComputeParameters(DiagnosticBag diagnost
                     diagnostics.Add(ErrorCode.ERR_ParameterIsStaticClass, this.locations[0], propertyType.Type);
                 }
 
-                parameters.Add(new SynthesizedAccessorValueParameterSymbol(this, propertyType, parameters.Count));
+                parameters.Add(new SynthesizedAccessorValueParameterSymbol(this, propertyType, parameters.Count, _property.SetterValueFlowAnalysisAnnotations));
             }
 
             return parameters.ToImmutableAndFree();
         }
 
+        internal override void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData> attributes)
```

Comment:
>SynthesizedAttributeData [](start = 114, length = 24)

It looks like we can easily relax this type to ```SourceAttributeData``` and return the instance we get from property.GetAttributes()

Reply:
Alternatively, to minimize code churn, we can create a thin wrapper.

---
In reply to: [295429868](https://github.com/dotnet/roslyn/pull/36444#discussion_r295429868) [](ancestors = 295429868)

Reply:
Or just transfer information to the new instance.

---
In reply to: [295431906](https://github.com/dotnet/roslyn/pull/36444#discussion_r295431906) [](ancestors = 295431906,295429868)